### PR TITLE
Add fix for nil state in InitializeFromProto

### PR DESCRIPTION
**Pull request by 0xKiwi**
**What type of PR is this?**
Bug fix

**What does this PR do? Why is it needed?**
This PR adds a fix and regression test for a nil state panic detected by fuzzit.

### CODE REVIEW

File: tools/genesis-state-gen/main.go [hunk 1]
```diff
@@ -6,6 +6,8 @@ import (
 	"io/ioutil"
 	"log"
 
+	"github.com/prysmaticlabs/prysm/shared/debug"
+
```
Comment (rauljordan) [inline tools/genesis-state-gen/main.go:10]:
```suggestion
```

File: tools/genesis-state-gen/main.go [hunk 2]
```diff
@@ -19,6 +20,7 @@ var (
 	sszOutputFile    = flag.String("output-ssz", "", "Output filename of the SSZ marshaling of the generated genesis state")
 	yamlOutputFile   = flag.String("output-yaml", "", "Output filename of the YAML marshaling of the generated genesis state")
 	jsonOutputFile   = flag.String("output-json", "", "Output filename of the JSON marshaling of the generated genesis state")
+	cpuProfile       = flag.String("cpuprofile", "", "Output filename for cpu profiling")
```
Comment (terencechain) [inline tools/genesis-state-gen/main.go:23]: why is this part of the PR?

Comment (nisdas) [reply tools/genesis-state-gen/main.go:23]: +1 , it has nothing to do with the pr

Comment (0xKiwi) [reply tools/genesis-state-gen/main.go:23]: Ah my bad, thanks for catching

**Pull request merged**